api/tests/test-router.js

```diff
@@ -78,6 +78,8 @@ o.spec("route", function() {
 					callAsync(function() {
 						o(root.firstChild.nodeName).equals("DIV")
 
+						o($window.location.pathname).equals("/a")
```

Comment:
Ah, right, makes sense :-)

Although, I actually don't think this test "works" because the new version of the test with the old version of the code still passes :-s (But based on real browser testing it fails)

(Found it, will make a PR)